core/util/makeConfig.js

```diff
@@ -100,6 +100,7 @@ function makeConfig (argv) {
 
   config.defaultMisMatchThreshold = 0.1;
   config.debug = userConfig.debug || false;
+  config.defaultIsSameDimensions = userConfig.isSameDimensions || true;
 
```

Comment:
This will always resolve to true.   Try `userConfig.isSameDimensions === false ? false : true`